RELEASE_PROCESS.md

```diff
@@ -38,16 +38,10 @@ Then you can copy and paste the commands below without worrying about mistyping
     ```
 
 4. Find the tag you just pushed [on GitHub](https://github.com/allenai/allennlp/tags) and
-click edit. Now copy over the latest section from the `CHANGELOG.md`. And finally, add a section called "Commits" with the output from this:
+click edit. Now copy over the latest section from the `CHANGELOG.md`. And finally, add a section called "Commits" with the output a command like the following:
 
     ```bash
-    git log `git describe --always --tags --abbrev=0 HEAD^^`..HEAD^ --oneline
-    ```
-
-    Or, if you're using fish,
-
-    ```fish
-    git log (git describe --always --tags --abbrev=0 HEAD^^)..HEAD^ --oneline
+    git log v0.9.0..v1.0.0rc4 --oneline
```

Comment:
I feel like this actually makes it more error prone since you can't just copy and paste this commad.. you'd have to edit it to change the tags.

And `git describe --always --tags --abbrev=0 HEAD^^` just prints the name of the previous tag.

Would doing

```suggestion
    git log `git describe --always --tags --abbrev=0 HEAD^^`..$TAG --oneline
```

make it more clear?

Reply:
Yes, substantially.  It would also reduce error, as the previous command depends on the commit you're at.